msteams-platform/messaging-extensions/how-to/enable-SSO-auth-me.md

```diff
@@ -0,0 +1,114 @@
+---
+title: SSO support for your messaging extensions
+author: clearab
+description: How to enable SSO support for your messaging extensions
+ms.topic: conceptual
+ms.author: lajanuar
+---
+
+# Single sign-on (SSO) support for messaging extensions
+ 
+**Single sign-on** support is now available for messaging extensions and link unfurling. Enabling **Single sign-on (SSO)** for messaging extensions silently refreshes the authentication token, which minimizes the number of times you need to enter your sign in credentials for Microsoft Teams.
+
+This document guides you on how to enable the SSO and store your authentication token, if required. 
```

Comment:
Can you link the article on adding single sing-on support for bots from this page? The prerequisites are essentially the same and they can follow along the steps over there. 

https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/authentication/auth-aad-sso-bots#:~:text=%20Develop%20an%20SSO%20Teams%20bot%20%201,token%20is%20a%20normal%20POST%20message...%20More